metabase/metabase
# don't run pr envs on external

### Description

Stops PR Env creation on external PRs. If the PR has the label `External` then PR Env will not be created.

### How to verify
Create a PR with the label `External`. Add the label `PR-Env`. Ensure the PR Env isn't created.
